main.go

```diff
@@ -141,6 +141,7 @@ func main() {
 	if err != nil {
 		// TODO: this doesn't seem to get called even when an error returns from a command, but maybe urfave is doing a non zero exit anyways? nope: https://github.com/urfave/cli/issues/610
 		fmt.Fprintf(os.Stderr, "ERROR: %v\n", err)
+		fmt.Fprintf(os.Stderr, "Note: the fn CLI command structure has changed, you may need to update your CLI\n")
```

Comment:
They've already updataed their CLI ! 

They may need to change the command they typed!

Reply:
oh dur, completely miss understood that one.

Reply:
@msgodf they would have to update it to even get this error message, so its just the command.

Reply:
It might be just good to give an example

"If you ran `fn apps list`, try running `fn list apps` instead"

Reply:
@JadeRedworth yes me too :) I misunderstood both that *and* @zootalures's suggestion!

Reply:
This is in the wrong place then as there's an `app.CommandNotFound` func anyway.